patches/@synthetixio__synpress-metamask@0.0.1-alpha.7.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/src/fixtures/metaMaskFixtures.ts b/src/fixtures/metaMaskFixtures.ts
+index cc2019e4baa64c8a7dee64398c09dd415169a17f..cda6d980723f3ce885d4fab99596a3e07384a65e 100644
+--- a/src/fixtures/metaMaskFixtures.ts
++++ b/src/fixtures/metaMaskFixtures.ts
+@@ -53,7 +53,7 @@ export const metaMaskFixtures = (walletSetup: ReturnType<typeof defineWalletSetu
+       // We don't need the `--load-extension` arg since the extension is already loaded in the cache.
+       const browserArgs = [`--disable-extensions-except=${metamaskPath}`]
+ 
+-      if (process.env.HEADLESS) {
++      if (process.env["HEADLESS"]) {
```

Comment:
is this patch actually required? seems like a linter difference

Reply:
This is an issue with Next.js build - added details to the PR description. As I check there is no certain issue related with similar environment on Synpress' GH issues, needed to patch this package